README.md

```diff
@@ -27,6 +27,112 @@ fastify.get('/another/path', function (req, reply) {
 
 ```
 
+### Options
```

Comment:
Can we link to the options in send docs instead? I’m not a fun of duplicating them.

Reply:
Sure. Would you like me to keep the descriptions for the Fastify options though?

Reply:
yes!